### ISSUE TITLE
Device support Bticino Living Now

### ISSUE THREAD

**Issue by zinoalex** [dresden-elektronik/deconz-rest-plugin#2707]
Hi guys, would be possible to expand the support to the Bticino Living Now devices?
I was able to include (only on channel 11 and after device's firmware upgrade) and use:

K4027C shutter controller (light in Phoscon)
F20T60A: DIN energy monitor mistakenly called Legrand DIN power consumption module (not visible in phoscon)
K4531C: power consumption module (light in Phoscon)
K4027CW: wireless shutter command (not visible in phoscon)
In all those devices the manufacturer is wrong reporting Legrand.

Unable to use:
K4003C: connected deviator
This one is recognized and included in the network after FW upgrade to v22 and channel 11 but it doesn't report the actual status. Toggle mode in deconz works. Any ideas to do that? I read a post about zigbee2mqtt and they were able to control this device.

Device
Screenshots

**Comment by Smanar** [comment]
Hello, xas my question here ^^ https://github.com/dresden-elektronik/deconz-rest-plugin/issues/2674#issuecomment-614096237

How are you using deconz for this device ?
The api haven't toogle fonction, it have only "on" and "off" ?

And it doesn't report status because this device is not in whitelist for binding. I can add it in the code (just one line to add).

To activate the reporting, waiting for code modification, you can use deconz

First make the bind
- Display the bind dropbox
- drag and drop the cluster 0006 from your device to the "source box"
- drag and drop the endpoint 01 (the title of the node") from the conbee to the "destination box"
- press "bind" button

Then enable attribute reporting
- display the cluster 0006 on/off in deconz
- on the table on the bottom, select the attribute 0x0000
- double click, press "read" and check values, 1 /300 is not bad value.

Or if you can compile the code, I can make modification now, but I need to understand how it work, because there is no real "on" and "off", if you use the output 2 it will be "off" but if you use the output 1 it will be "on".

And if you are using a not connected deviator on the same circuit, it can be on and off without action on the device.

**Comment by zinoalex** [comment]
Sorry Smanar I've missed the post.
I'm using the GUI accessible via VNC latest release (.75) in docker with Conbee II.
![toggle](https://user-images.githubusercontent.com/54627059/80313059-1654b000-87e9-11ea-9e32-de4fbe06c591.jpg)
I have these functions: Off, On and toggle. With off nothing happens, with On and Toggle the device switches between pin 1 and 2.

The result after you instructions:
![binding](https://user-images.githubusercontent.com/54627059/80313118-5a47b500-87e9-11ea-8070-afc67f326200.jpg)
the value stays "false" either with power on pin 1 and 2

**Comment by Smanar** [comment]
Ok, so if the value stay false, it will be a problem, I will check z2m to see If I found more information

But can you use the device ATM with the API ? There is no toogle command on it, how are you using the device ? If you seend multiple "on" command it toogle the output ?

**Comment by Smanar** [comment]
Ok so I have found something
>The device has a single command button that commutes the phase(L) between the 2 outputs (two-way-switch/va-et-vient) so one of the 2 output is always 'ON', and a little reset/paring button (not accessible when the cover is mounted/during normal operation) + LED. As you can see the device always reports back the state of OFF (but the light is ON)

But 
>Both my K4003C are responding correctly to ON OFF commands and reports back correct state **even when switching ON/OFF from non smart switches** (two way wiring)

So I think the device can check the consumption or current presence, are you making try with a bulb or other thing that consumme current ?

**Comment by zinoalex** [comment]
Update: I connected the device with a bulb and another deviator and actually works correctly. Now the ON/OFF status changes between true and false, so it must have a kind of consumption meter inside.

**Comment by Smanar** [comment]
Realy nice hardware, seriously, was a good idea to use something hardware to guess if it's on or of.
IDK why we haven't it in france, perhaps because we don't use neutral for switch.

So, nothing to change ? I just need to enable the binding/report automaticly ?

**Comment by zinoalex** [comment]
I think so, anyway I connected all the 4 deviators that were previously connected to the Bticino gateway and everything is working great with Home Assistant, the trick was to connect a load in order to change the on/off flag.
yes is actually a very smart piece of hardware, the only limitations are that you need the Bticino Living Now series which is quite new and a neutral wire.
Merci anyway Smanar

**Comment by Smanar** [comment]
All is already working ?
You have done the binding/report manually, or they don't need it for state return ?

**Comment by zinoalex** [comment]
No need for manual binding

**Comment by Smanar** [comment]
Ok thx, so I let the code as it.

**Comment by Mimiix** [comment]
It seems this issue is inactive / resolved. If it is not, please re-open or create another issue!

**Comment by lambu76** [comment]
Hi all, I have a strange problem with a BTicino connected deviator (N4003C) and Conbee II.

I have a fresh installation of deCONZ  (last version V2_05_86).

Once configured to use the channel 11 and reset the deviator to default factory I can see inot the zigbee network, connected t the Conbee II,  but after a few minutes the binding is lost and it's not anymore possible to add it.

let me underline between the automatic adding into the network and the lost of bidning I didn't perform any action...just waiting for the lost of bidnding.

Any ideas ?

thx

**Comment by Smanar** [comment]
Yep, probably a bad inclusion.
Legrand device join easily a network but if the procedure is not respected they leave immediatly after.

What if your inclusion procedure ? remember you need to use the hidden button not the deviator itself.

**Comment by lambu76** [comment]
mmmmm.....maybe I missed something...I'm very rookie on this topic.

Could you please remember which is the proper procedure ?

Personally I just pressed the reset button in order to restore default factory settings into the deviator, but nothing else.

It has been joined to the Zigbee network.

thx

**Comment by Smanar** [comment]
I have only Legrand device on my side, not Bticino but for me it's 

Long press to reset, ignoring the first blink, , release, then short press.
You have a led blinking, you need to have green color.

**Comment by lambu76** [comment]
ok thx I will try.

Just a quick question.

Do you have the original gateway in place ?

Currently I'm trying such kind of test without the Legrand/Bticino gateway.

thx

**Comment by Smanar** [comment]
I have it, but in a box, not connected.
But it's not the same procedure, it s easier with the original gateway.

**Comment by lambu76** [comment]
unfortunately with BTicino if I take pressed button for reset I have the first blink, but after that if I release and repress the button nothing else happen.

I need someone else who have already integrated a BTicino component.

**Comment by lambu76** [comment]
> No need for manual binding

Hi, can I kindly ask you your help ?

thx

**Comment by Smanar** [comment]
You need to ignore the first blink, and keep pressed.

If @zinoalex have another procedure ?

**Comment by zinoalex** [comment]
Hi lambu,
first of all you need to pair your deviator with the original Bticino gateway in order to update the firmware (at least v22, it takes quite a few time), than reset to factory pressing the button (keep it pressed). Switch off the bticino gateway and all the other zigbee gateways (Hue, ecc) except the conbee II. Turn on the discovery as a light and start to press the deviator reset button. Remember that is not immediate, vnc page is very useful to check if it is included or not. Most of my network problems were from the double zigbee network (hue and conbee) + wifi channel overlapping. Now i use 3 deviator, 3 blind controller and a couple of scenery button since months with no issues at all. Anyway check the channels https://community-assets.home-assistant.io/original/3X/2/f/2fa03d15dd0a963475c38cb03d776b8e8750c9b5.png

**Comment by Smanar** [comment]
But it works too without the gateway ?
For Legrand not updated device are easier to include. I can send too the firwmare to update it with deconz but need to check if I have it, and the actual deconz version have some problem with OTA.

**Comment by Smanar** [comment]
Yeah ^^, I m already on it > https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3532

By hazard can you compile the code to test it ? The other user  don't have unix machine.

**Comment by lambu76** [comment]
unfortunately I installed deconz on Windows and Hass.IO as virtual appliance by downloading a VHDX virtual disk.

I took a look to the post mentioend by you, but I'm not getting if you are mentioning my same identical problem or something else.

thx

P.S.
if you are italian you can contact me on my mail:

user: lambu76
domain: gmail.com

**Comment by Smanar** [comment]
Ha yep sorry, the discution have started on discord, but it s a similar issue, if you want to make it working you need to use the "open" command, HA use "bri" and this command is buggy witht his device.

I will make the PR in blind mode, but I think it will be nought, I have mimic the other shutter device.

**Comment by lambu76** [comment]
I'm a little bit lost but happy to know that can work...could you please support me in order to instruct HA to send the proper command?
Which conf file I have to update?

Thx a lot

**Comment by Smanar** [comment]
I don't know at all HA.
But you can't update your files on Hassio to test the new version.

You need to change the command done by the app to make try with the not corrected version, or use direclty the API with Curl for exemple.

**Comment by lambu76** [comment]
Where did you find that the reason for this bug is a wrong command send by HA ?

Maybe I could find someone else in some community to help me with HA, since unfortuantely you are not using it.

thx

**Comment by Smanar** [comment]
no it s not HA fault.
HA send 2 commands in one, "bri" and "on" or "lift" and "open", and this device have one of them reversed.

I don't remember, but for one HA use only 1, but both for the other side, so the device faild.

But it s not HA fault, the command need to be reversed in deconz code, or as workaround, you can edit the command send by HA, waiting for the correction.

**Comment by lambu76** [comment]
thx for the explanation....I'm trying to find a way to do that....I'm pretty rookie on such topics

**Comment by Smanar** [comment]
I m sorry but I don"t know at all HA, but if you find the command used by it, just remove the "bri" or "lift" part.

But the modification will be in the next deconz version I think, not a critic one

**Comment by testorp** [comment]
hello i also have a L4003C and I'm trying to pair with deconz but no way am I able to, it never comes up on the ZigBee network, even though I'm able to use it with it's own gateway L4500C

any suggestions how to do so?

this is what is running on Phoscon
![image](https://user-images.githubusercontent.com/44940599/101516975-723b0600-3980-11eb-84ed-7e0442d67e8d.png)

Any help aprreciated

PAOLO

**Comment by Smanar** [comment]
Ha, I think I miss this one.
Do you have the model id of the device visible in deconz (on the GUI) ?

**Comment by testorp** [comment]
No deCONZ is not able to pick it up

**Comment by Smanar** [comment]
The L4003C is the same device than the K4003C ?
Wich one inclusion procedure are you using ?

- Long press (around 10s)
- keep pressed on first blink
- Led go red fixe
- short press
- if LED don't go green, retry.

**Comment by testorp** [comment]
@Smanar L4003C is Bticino Living Light series
![image](https://user-images.githubusercontent.com/44940599/101544563-2e5af780-39a6-11eb-8d67-b39211afc3c4.jpeg)

K4003C is Bticino Living Now series
![image](https://user-images.githubusercontent.com/44940599/101544545-2733e980-39a6-11eb-9463-b2f98acf1bcc.jpeg)

**Comment by testorp** [comment]
@Smanar don't know what I've done but in the end I've been able to get the switch in deCONZ, more or less what you have described.
what you are explaining is that red led fixed means the switch must be in factory reset, then pressing the reset switch again it goes green led and it's able to pair

Bticino has 2 lines of connected devices Living Light and Living Now see picture below

![image](https://user-images.githubusercontent.com/44940599/101638045-d9fb5a80-3a2d-11eb-86b9-f6ec24634dfe.png)

here is a document that explains it from Bticino
[91735.pdf](https://github.com/dresden-elektronik/deconz-rest-plugin/files/5666274/91735.pdf)

**Comment by Smanar** [comment]
Yeah the deviator is not used in my country, but rely nice device too.
But I haven't all model id for bticino, for light it s not a problem because they are working natively, but you can miss some feature on other like power mesurement, but the good news is legrand repect zigbee rule, so every time it s just a "1 line edition" modification.

**Comment by testorp** [comment]
@Smanar Any chance of having the switches identified as switches and not light?

Thx
Paolo

**Comment by Smanar** [comment]
Nope this is not a bug, it s normal.
Battery switches are sensor (sub section switch).
Wired switches are lights.

You send order to the deviator to turn on the light, you don't use this position to make something else.

**Comment by testorp** [comment]
Is it possible for me to change this behavior in code?

If I use the gateway from Bticino they appear as switches in HomeKit

**Comment by Smanar** [comment]
It depend of your third app, but this device realy need to be managed as a light, not a switch.
So I don't think you can change that on "core" perhaps on the third app that display it ?

**Comment by testorp** [comment]
Can you please explain why?

Please, trying to understand.

I'm using Phoscon and Homebridge-Hue to go directly to Apple HomeKit so I can't change anything

**Comment by Smanar** [comment]
For exemple, you can use a group light, and turn all the light in it in same time, you put in it the deviator, but not a battery switch.
Wired switches are working like light. You can use a battery switch to turn on the deviator, but not the reverse.

**Comment by testorp** [comment]
Ok the point is I want to use both connected switches and connected lights

What you are saying makes more sense if lights are not connected

**Comment by lambu76** [comment]
Hi guys, I'm back on this topic after a long time.

After last updates from Deconz BTicino shutter works fine in opening and closing  direction...very good.

Now I have two questions.

1) How can I force the opening or closing to a specific position ? currently I can only drive the full opening (value 100) or full closing (value 0). Any other values are not set and in case I open only paritally a shutter manually the values grabbed is always 50.

2) is it possibile to have the light embedded into the device swithced on ? Currently is swtiched on (red color) only when it's not connected to zigbee network, then it becomes for  short while on green when I'm joinning the network and then it's definitely switched off.

thx to all

**Comment by Smanar** [comment]
@lambu76 I m interested by your version ^^ https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3876#issuecomment-747613679
There is soon a new version for this device, don"t want to broke something.

1- this device don't support set level command
2 - https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3876#issuecomment-747447209

**Comment by zinoalex** [comment]
@Smanar  yes it's true but then I don't know how to interface it with home assistant. I use the dockerized version of deconz.

@lambu76 
1: works fine with bticino app after "calibration" but there is no way to do it in deconz, I used timers with home assistant (not a very elegant way but it works)
2: option 2: true

**Comment by lambu76** [comment]
> @Smanar yes it's true but then I don't know how to interface it with home assistant. I use the dockerized version of deconz.
> 
> @lambu76
> 1: works fine with bticino app after "calibration" but there is no way to do it in deconz, I used timers with home assistant (not a very elegant way but it works)
> 2: option 2: true

1: damn.....can you share me some tips about how to use timers ? Not elegant solution but working is better than NO solution

2: I'm getting "WRITING FAILED" on some devices (shutter or normal lights always bticino) and some other not...any ideas ?

**Comment by lambu76** [comment]
Let me rephrase.

I'm getting "WRITING FAILED" on ALL my shutters L4027C, but Option 2 is working fine on on light device L4003C.

Firmware on both devices is 00b...as per Phoscon webapp indication.

Ideas ?

**Comment by zinoalex** [comment]
In my case It changed after firmware upgrade with the original gateway.
About timers:
take the time for the shutter to completly close than make an automation
with the half of the seconds for example for the 50% position and so on.
you can use home assistant automations or either node red. you can also
make an input boolean switch to actuate a script

Il dom 3 gen 2021, 12:56 lambu76 <notifications@github.com> ha scritto:

**Comment by lambu76** [comment]
@zinoalex 
got it for the timers....I will try it.

For the upgrade, my problem is that I don't have an original gateway....is there any way to upgrade to latest firmware without it ?

Which is the last firmware version available ? Currently I have 00b on all my BTicino devices.

If I have an original gateway is there any way to upgrade to latest firmware without deconnecting devices from deconz, connecting to original gateway and the reconnecting to deconz ?

**Comment by Smanar** [comment]
Someone with the same version than you had exactly same problem.
Led not working and impossible to use deconz for update, the upgrade server was locked to 0x0000000

![notworking](https://user-images.githubusercontent.com/20152487/103480099-897be080-4dd2-11eb-8357-f2d5cda152a5.png)

**Comment by lambu76** [comment]
So the original gateway is neeed....damn, but it would be not a big deal if I can use it without removing the evice from zigbee network managed by deconz...but I don't know if it's possible.

So my question is....supposing to have a BTcino device connected to zigbee network managed by deconz device, whihc are the steps to upgrading to last firmware by using a the original gateway ?

**Comment by Smanar** [comment]
Sorry I m seing only 2 solutions
- find someone with the bticino firmware and use deconz, but if you are in same situation than my previous post, it will not work
- disconnect your device from deconz network to connect it to bticino one, and just wait 24 h.

**Comment by zinoalex** [comment]
I had to remove the device from deconz and then repair It with original
gateway, It takes more or less one hour to do the upgrade. I was unable to
update the firmware with deconz even if my server was not 0x000

Il dom 3 gen 2021, 15:50 Smanar <notifications@github.com> ha scritto:

**Comment by zinoalex** [comment]
As far as i remember the switch has 4 pins, meaby with some reverse
engineering and an Arduino programmer we are able to download the firmware

Il dom 3 gen 2021, 16:15 Alessandro Zinelli <zinellialex@gmail.com> ha
scritto:

**Comment by Smanar** [comment]
You can just get them using this method https://github.com/pipiche38/Domoticz-Zigate-Wiki/blob/master/en-eng/Retreiving-Legrand-Firmware.md

But it have a little changed ATM, for the file name.

**Comment by muchasuerte** [comment]
hello @lambu76 , 

did you set any specific value on network for channel mask or APS ack or Security mode?

I have a Relè 3584C but I cannot include it, I have not found any way to get a green led.

![immagine](https://user-images.githubusercontent.com/1520216/107703361-d5685200-6cbb-11eb-8b43-73200d0f8da8.png)

**Comment by muchasuerte** [comment]
I did some test by setting the channel mask to 11 and 15 and also checking APS Acks and now I can see the device, the problem now it that I lost all other devices :(

![immagine](https://user-images.githubusercontent.com/1520216/107709126-fda87e80-6cc4-11eb-852a-a668f3fbcacf.png)

![immagine](https://user-images.githubusercontent.com/1520216/107710186-cd61df80-6cc6-11eb-9c65-f8f8ad428fc3.png)

**Comment by Smanar** [comment]
Legrand use channel 11 (channel locked), so I think Bticino do same.

**Comment by muchasuerte** [comment]
ok, so it is the problem, is risky change the channel? at the moment I have restored my network setting the ch 15.

is there any risk on change the channel from 15 to 11 to an already configured network?

**Comment by Smanar** [comment]
Theoretically no, in reality .....
But you can make a backup in phoscon first, you can have to power cycle your router, and some end devices can realy don't like that at all, but it s possible.

**Comment by muchasuerte** [comment]
the switch from ch15 to ch11 was quite fast or at least for now it works and I was able to pair it with 5 sec reset + 1 push  

@Smanar thank you for the support, the key point was the channel

**Comment by lambu76** [comment]
Hi guys, I want to share with you a corious behavior of my BTCino devices (roller shutter and switches).

Sometimes (I don't know the frequency...I just saw it two time) all embedded leds turned in green color, they stay as is for some minutes and they reverted back in blue color.
When they are green the devices still working fine.

First time I discovered more than 1 month ago at 7.00AM when I waked up, last time last week at 10.00PM .

In both case I didn't perform any kind of action of devices, deconz or HA configuration nor in that specific moment nor in the past hours.

Any ideas ?

**Comment by Smanar** [comment]
Yep, take a look in log if you have not a "permit join" command at this moment.

**Comment by lambu76** [comment]
could you please teach me where I can find the log ?
Currently I'm using the Deconz add-on in HassIO.

**Comment by Smanar** [comment]
I don't know at all HASSio, but from my memory you can access to deconz GUI ? Then after just go to deconz/help/log
Else https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/deCONZ-debug-flags

**Comment by lambu76** [comment]
here what I can see around the moment of the led went in green color.

![image](https://user-images.githubusercontent.com/73443319/111879635-2e589380-89a8-11eb-8fd6-2e28feaaca7d.png)

**Comment by Smanar** [comment]
Its exactly that ^^.

For information deconz send regulary this command, but with duration = 0.
So I don't think it s from deconz, If you enable http I thnik you will be able to see if a third app is making the request.
But It strange too you have a websocket disconnection just before.

**Comment by lambu76** [comment]
ok thx.

I flagged HTTP checkbox and I will it .... I will let you know next time I will see green color.

Other question....do you know if there is any new about updating Bticino device firmware without their own gateway ?

**Comment by Smanar** [comment]
Some user had as date "somes weeks", on my side I had "somes months".
But you have a problem with a device that realy need an update ?

**Comment by lambu76** [comment]
My hope is that with a new firmware it will be possibile to set a specific percentage for opening/closing roller shutter.
Maybe I'm wrong.
Currently I have all my roller shutter connected to the BTicino devices but I can only set 0%, 100%, any other value is not supported and if I open just a bit or mostly the roller shutter for the decive is open at 50%

**Comment by Smanar** [comment]
It don't depend of the switch but the cover. Your cover don't support position report, so you can only use a timer, can't be precise.
if your shutter need 10s to be open, just send a "open" order and a "stop" 5s after ?

**Comment by lambu76** [comment]
ooohhh...so it means that with the BTicino gateway such functionality is implemented by setting specific timer per each roller shutter.

Ok...my bad.

What about the led light for the roller shutter switches ? is it possibile to switch on with a new firmware by using Deconz or not yet ?
I really would like to have them on instead off as now.

**Comment by Smanar** [comment]
For the LED it s a good question.
No answer from my side, lot of users have tried different firwmare, same result, the knowed attributes not working, so there is an attributes not found yet, or a bug in the device.

**Comment by LucaBors** [comment]
Hi guys, did it with a ‎K4003C, all good so far and it's connected correctly to my home assistant. I'd like to use the led inside (the blue one) so that, when light is off led is on. I've found something that can work but it's based on mqtt. 
– service: mqtt.publish
          data:
            topic: “zigbee2mqtt/FRIENDLY_NAME/set”
            payload: “{\”led_when_on\”: \”ON\”, \”permanent_led\”: \”OFF\”}”

there's a way to "traslate" it natively to deconz? 

Thank you

**Comment by Smanar** [comment]
Yes it's probably already in the XML file.
But only possible if you have access to the GUI ?

Its on the Legrand cluster, it s called "option 1" or "option 2",

**Comment by LucaBors** [comment]
oh, thank you! this run perfect! :)

**Comment by s1gmund80** [comment]
Hi all guys, since a couple of month the Bticino DIN power consumption module always stop working after 24h from the host restart. 
To let it work again I need to restart the Host (core or supervisors are not enough), but It will last only 24h
Anyone else has the same behaviour? Any suggestion?

I'm on HA (core-2022.3.8, supervisor-2022.05.0, Home Assistant OS 7.6) with Phoscon/ Deconz (2.15.03 / 21/4/2022 - 26720700)

**Comment by Smanar** [comment]
And other zigbee module still working ? Only this one stop working ?
Can you chedk the device json when it have stoped working ? (in phoscon / help / API Information /sensors )

**Comment by s1gmund80** [comment]
Yes, every other zigbee device works fine. As soon as it stops again i’ll post the json (i still need to wait few hours)

**Comment by s1gmund80** [comment]
> And other zigbee module still working ? Only this one stop working ?
> Can you show the device json when it have stoped working ? (in phoscon / help / API Information /sensors )

Here the json in phoscon as "switch": 

{
    "config": {
        "group": null,
        "on": true,
        **"reachable": true**
    },
    "ep": 1,
    "etag": "247145422c578308c66b138fa37cc61d",
    "lastannounced": null,
    "lastseen": "2022-05-09T08:27Z",
    "manufacturername": "Legrand",
    "mode": 1,
    "modelid": "DIN power consumption module",
    "name": "DIN BTicino consumo",
    "state": {
        "buttonevent": null,
        "lastupdated": "none"
    },
    "swversion": "0036",
    "type": "ZHASwitch",
    "uniqueid": "00:04:74:00:00:a1:5c:02-01-0006"
}

And here as sensor: 
{
    "config": {
        "on": true,
        **"reachable": false**
    },
    "ep": 1,
    "etag": "f6b39bb442737488f010b5652eb04a03",
    "lastannounced": null,
    "lastseen": "2022-04-03T08:12Z",
    "manufacturername": "Legrand",
    "modelid": "DIN power consumption module",
    "name": "Power 8",
    "state": {
        "lastupdated": "2022-05-09T08:30:51.484",
        "power": 733
    },
    "swversion": "0036",
    "type": "ZHAPower",
    "uniqueid": "00:04:74:00:00:a1:5c:02-01-0b04"
}

The sensor says it is not reachable, but in the stream tab I can see the event sent.

**Comment by Smanar** [comment]
Ha yes, we can see the "lastupdated" in date but only the ZHASwitch have "reachable" to true.

```
        "reachable": false
        "lastseen": "2022-04-03T08:12Z",
        "lastupdated": "2022-05-09T08:30:51.484",
```

"lastseen" is not updated, It remeber me an old issue.

**Comment by Smanar** [comment]
No, it s ok.
I have enought information for the moment.
Manup seem interested by this issue, so I prefer let him search for it, for me it s a problem in DDF core, and I don't know a lot this part.

Will see after if a DDF can help or not.

**Comment by s1gmund80** [comment]
Should I open a new issue rather than monitoring this closed one?

**Comment by Mimiix** [comment]
I'll reopen.

**Comment by Smanar** [comment]
Your issue is almost the same than here https://forum.phoscon.de/t/since-migrating-deconz-plugin-from-2-14-1-to-2-15-3-heiman-hs2wd-e-siren-sensors-become-note-reachable-after-a-few-hours/1951

Manup have updated the firwmare since, but I don't think he had enought time to check the issue.

**Comment by Smanar** [comment]
Your issue is probalby solved with this PR https://github.com/dresden-elektronik/deconz-rest-plugin/pull/6106

**Comment by eternum79** [comment]
Has anyone been able to solve the problem of the open / close percentage not recognized by the k4027c device?

**Comment by mariomaz87** [comment]
> Has anyone been able to solve the problem of the open / close percentage not recognized by the k4027c device? deconz on homeaasistant. Thanks

Did you? I have been struggling with this today. I have successfully added the device on Deconz\Phoson but I can only open or closed it. Thanks!

**Comment by Smanar** [comment]
Are you sure your device support it ?
You can try direclty with the GUI on the covering cluster.

**Comment by mariomaz87** [comment]
> > > Are you sure your device support it ? You can try direclty with the GUI on the covering cluster.
> > > If yes, pls open a new issue with all the needed information to make a new DDF.
> > 
> > 
> > Yes, using the GUI it works. I opened the issue thanks!
> 
> does it mean you are able to open cover to a specific percentage ?

Yes, it’s working as expected.

**Comment by Smanar** [comment]
I think the next will be here https://github.com/dresden-elektronik/deconz-rest-plugin/issues/7390#issuecomment-1828297609
